test/integration/src/test/java/io/pravega/test/integration/ControllerMetricsTest.java

```diff
@@ -209,23 +209,23 @@ public void streamMetricsTest() throws Exception {
         //Put assertion on different lines so it can tell more information in case of failure.
         Timer latencyValues1 = MetricRegistryUtils.getTimer(getTimerMetricName(CREATE_STREAM_LATENCY));
         Assert.assertNotNull(latencyValues1);
-        Assert.assertTrue(iterations <= latencyValues1.count());  //also system streams created so count() is bigger
-
+        Assert.assertTrue("Iterations " + iterations + ", Count " + latencyValues1.count(),  //also system streams created so count() is larger
```

Comment:
`AssertExtensions.assertLestThanOrEqual` will output the actual values in the error message.